models/ed/R/model2netcdf.ED2.R

```diff
@@ -103,18 +103,18 @@ model2netcdf.ED2 <- function(outdir, sitelat, sitelon, start_date, end_date, pft
                                outdir, start_date, end_date, 
                                pft_names)
     }
-    
+    #length(out_list[["-T-"]]); length(out_list[["-E-"]])
     
     if (y == strftime(start_date, "%Y")) {
-      begins <- as.numeric(strftime(start_date, "%j")) - 1
+      begin_date <- as.Date(strftime(start_date))
     } else {
-      begins <- 0
+      begin_date <- base::as.Date(paste0(y,"-01-01"))
```

Comment:
If we're going to use `base::as.Date`, we may as well be consistent, if only for clarity. I'm assuming these are supposed to be the same.